src/pages/products/components/ProductCard.jsx

```diff
@@ -10,21 +13,28 @@ function ProductCard({
 		badge,
 	},
 }) {
+	const { dispatch } = useCartWishlist();
 	let discount = Math.floor(((actualprice - newprice) * 100) / actualprice);
```

Comment:
can be converted to const. If it is not reassigned.

Suggestion: Also you can extract this into utils function as it has been used in product card and product card horizoontal

Reply:
Will do, thank you!